bcda/service/service.go

```diff
@@ -456,6 +458,16 @@ func (s *service) GetJobPriority(acoID string, resourceType string, sinceParam b
 	return priority
 }
 
+func (s *service) GetACOConfigForId(cmsId string) (*ACOConfig, bool) {
```

Comment:
thanks for this, I think other parts of the codebase can reuse this function.  mind adding a docstring?